editor/plugins/script_editor_plugin.cpp

```diff
@@ -3892,7 +3892,7 @@ ScriptEditor::ScriptEditor() {
 		vbc->add_child(disk_changed_list);
 		disk_changed_list->set_v_size_flags(SIZE_EXPAND_FILL);
 
-		disk_changed->connect("confirmed", callable_mp(this, &ScriptEditor::reload_scripts));
+		disk_changed->connect("confirmed", callable_mp(this, &ScriptEditor::reload_scripts).bind(true));
```

Comment:
Changes to the `gdscript_text_document.cpp` file in #66405 have been reverted in #66867, so #66405 is actually broken. I intend to synchronize the changes via a signal (#66880). Subsequent PRs will break some compatibility and have not been merged.


```suggestion
		disk_changed->connect("confirmed", callable_mp(this, &ScriptEditor::reload_scripts).bind(false));
```

This should be changed to `false` if we don't plan to restore the rest of #66405.